api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/GetConfiguredEndpointsResponseApiModel.cs

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models {
+    using System.Runtime.Serialization;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Result of GetConfiguredEndpoints method call
+    /// </summary>
+    [DataContract]
+    public class GetConfiguredEndpointsResponseApiModel {
+
+        /// <summary>
+        /// Service result in case of error
```

Comment:
>Service result in case of error [](http://example.com/codeflow?start=12&length=31)

the comment seems not related to the method

Reply:
thanks!